src/pages/staticPages/visualize/Visualize.tsx

```diff
@@ -6,9 +6,61 @@ import { PageLayout } from 'shared/components/PageLayout/PageLayout';
 import './styles.scss';
 import styles from './visualize.module.scss';
 import { getNCBIlink } from 'cbioportal-frontend-commons';
+import { getCustomButtonConfigs } from 'shared/components/CustomButton/CustomButtonServerConfig';
 
 @observer
 export default class Visualize extends React.Component<{}, {}> {
+    /**
+     * Display the 'visualize_html' data associated with serverConfig.custom_buttons_json
+     * @returns JSX.element
+     */
+    customButtonsSection() {
```

Comment:
We display one item per defined "CustomButton". Letting the config define the full HTML is not safe, so instead we follow the same pattern established by OncoPrinter and MutationMapper. ASIDE: you could always change those to be config-driven using the same "visualize_*" properties.